includes/admin/class-wc-admin-profile.php

```diff
@@ -171,8 +178,10 @@ public function add_customer_meta_fields( $user ) {
 								</select>
 							<?php elseif ( ! empty( $field['type'] ) && 'checkbox' === $field['type'] ) : ?>
 								<input type="checkbox" name="<?php echo esc_attr( $key ); ?>" id="<?php echo esc_attr( $key ); ?>" value="1" class="<?php echo ( ! empty( $field['class'] ) ? $field['class'] : '' ); ?>" <?php checked( (int) get_user_meta( $user->ID, $key, true ), 1, true ); ?> />
+							<?php elseif ( ! empty( $field['type'] ) && 'button' === $field['type'] ) : ?>
+								<button id="<?php echo esc_attr( $key ); ?>" class="button <?php echo ( ! empty( $field['class'] ) ? $field['class'] : '' ); ?>"><?php echo esc_attr( $field['text'] ); ?></button>
```

Comment:
Your code is not the only part guilty of doing it (I notice a few in here do the same thing) but there is a missing escape on the classname. Since this can be user-defined, it should be escaped. `esc_attr` should suffice - would have suggested we add `sanitize_html_class` but this will break if someone has used multiple classes separated with spaces for the arg.